docs/guides/job-limits.mdx

```diff
@@ -8,41 +8,21 @@ description: Job limits imposed by the job validation service for jobs sent to I
+Certain primitive options increase the circuit size. The described limits are checked *after* the expected increase in circuit size. In particular, these options increase circuit size:
+
+- Dynamical decoupling and twirling introduce additional single-qubit gates.
+- Gate-folding ZNE introduces additional two-qubit gates. The number of two-qubit gates is multiplied by the sum of noise factors requested in gate-folding ZNE.
 </Admonition>
 
 <span id="max-shots"></span>
 ## Maximum executions
 
-At most, **5 million** executions are allowed.  The number of executions is the number of circuits times the number of shots, where the circuits are those generated after PUB elements are broadcasted.
+At most, **10 million** executions are allowed.  The number of executions is the number of circuits times the number of shots, where the circuits are those generated after PUB elements are broadcasted.
 
 For example, if you have a PUB with one circuit, observables with shape (1, 6), and parameters with shape (4, 1), this would render $4 \times 6 = 24$ circuits (or fewer, if some observables commute). If you requested 2,000 shots, then the total number of executions is $24 \times 2,000 = 48,000$.
 
-<span id="per-qubit"></span>
-## Maximum number of low-level instructions per qubit
-
-The service permits up to **26.8 million control-system instructions per qubit**. This ensures that the user circuits fit within the control system's instruction memory.  The following table describes how the system translates instruction set architecture (ISA) circuit instructions to control system instructions when calculating this limit.
-
-| Instruction | Count |
-|-------------|-------|
-| `rz`          | 1     |
-| `delay`       | 1     |
-| `sx`          | 2     |
-| `x`           | 2     |
-| `cx`          | 5     |
-| `cz`          | 5     |
-| `ecr`         | 5     |
-| `measure`     | 10    |
-| `reset`       | 17    |
-| `init`        | 50    |
-
-<Admonition type="note">
-  This table captures the heuristic used in validation and does not reflect the exact number of instructions used to implement an operation.
-</Admonition>
-
 <span id="2-qubit-limit"></span>
-## Maximum number of two-qubit gates per job
+## Maximum number of two-qubit gates per circuit
 
+The maximum number of two-qubit gates per circuit is 400,000 for static circuits and 100,000 for dynamic circuits.
 This ensures that the job can be manipulated within the memory limits of the low-level software stack. The precise limit depends on the options selected in the Sampler or Estimator query. However, if a job has fewer than 10,000 two-qubit gates, it will pass validation. Some options might allow more, and the service will calculate the limit allowed for the user-selected options and communicate that should your job be rejected in validation.
```

Comment:
We don't need the wording "The precise limit depends on the option...". It's basically trying to convey the same message as the Note - that a limit of 400k gates doesn't mean you can actually pass in that many, given extra gates might get inserted.

Reply:
Ok! To be sure I understand: I should delete everything in that paragraph starting with "The precise limit depends on..."? Or only that single sentence?

Reply:
You can check my work [in this commit](https://github.com/Qiskit/documentation/pull/2768/commits/312394780bc6b51f18b5e28b7670095ee4235857)

Reply:
@jyu00 If you can check this for me - and confirm they've added support for gen3 to dedicated systems - I can merge today. Thanks!